app/views/projects/show.haml

```diff
@@ -49,7 +49,7 @@
           %tr
```

Comment:
@eanders am I correct in reading that this file is not yet "ACL-ready"? On line 108 below it appears to globally check `can_edit_projects?`, rather than checking if the user can edit _this_ project. (Maybe the correct check happens when the action is performed, but when deciding whether to show the button?). Same goes for the project delete button.

Reply:
This question is unrelated to the bug. It's just something I noticed.

Reply:
The view is probably not fully updated, but the controller should be.  As in you shouldn't be able to get to the page unless you have access setup correctly ACL or no.